toc/working-groups/app-runtime-platform.md

```diff
@@ -0,0 +1,181 @@
+# App Runtime Platform: Working Group Charter
+
+## Mission
+
+Provides operational components for the CF App Runtime, including those for application build, application execution, ingress and app-to-app routing, and aggregation of application logs and metrics.
+
+
+## Goals
+
+- End-user platform teams have reliable, performant, and well-documented system components to provide core CF App Runtime capabilities on designated infrastructure platforms, such as BOSH and Kubernetes.
+- Community contributors can build against core CF App Runtime system components via stable, well-documented APIs.
+- Community contributors can integrate tested CF App Runtime components into community reference deployments.
+
+
+## Scope
+
+- Develop system components that support core CF App Runtime capabilities, including building app artifacts from source code, running artifacts as apps, routing traffic to apps and between apps, and aggregating logs and metrics from applications for end-user consumption.
+- Maintain public roadmaps for the CF App Runtime component systems above and ensure that system component development matches roadmap intent.
+- Align component development to the priorities of App Runtime end users via collaboration with other Working Groups.
+- Provide community contributors with tooling and reference pipelines needed to build, test, and release App Runtime system components.
+- Collaborate with other Working Groups to ensure that App Runtime components are integrated regularly into the community reference deployments.
+- Maintain documentation for system component APIs, including which API groups are stable for use or are experimental and which API groups are intended for end-user internal use.
+- Ensure upgrade pathways for system components, with disruptive changes communicated clearly.
+
+
+
+## Non-Goals
+
+- Be responsible for these functional domain areas outside of the CF App Runtime system.
+
+
+## Proposed Membership
+
+Technical and Execution Lead: @ameowlia
+
+### Approvers by area
+
+Diego:
+- @acrmp
+- @aminjam
+- @Benjamintf1
+- @geofffranks
+- @IvanHristov98
+- @jrussett
+- @pivotalgeorge
+- @reneighbor
+- @selzoc
+
+Eirini:
+- @danail-branekov
+- @gcapizzi
+- @georgethebeatle
+- @kieron-dev
+- @mnitchev
+
+Garden Containers:
+- @aemengo
+- @danail-branekov
+- @gcapizzi
+- @georgethebeatle
+- @kieron-dev
+- @mnitchev
+
+Logging and Metrics:
+- @acrmp
+- @aminjam
+- @Benjamintf1
+- @geofffranks
+- @jrussett
+- @pivotalgeorge
+- @reneighbor
+- @selzoc
+
+Networking:
```

Comment:
I'm nominating Patrick Lowin (@plowin) from SAP to be an approver in the networking area. He is a current maintainer of the haproxy bosh release.

Reply:
Done!